merkletree/merkletree.go

```diff
@@ -13,8 +15,8 @@ var (
 )
 
 const (
-	EmptyBranchIdentifier = 'E'
-	LeafIdentifier        = 'L'
+	EmptyNodeKey = 'E'
+	LeafNodeKey  = 'L'
```

Comment:
Nitpicking: We would need this naming to be consistent. I liked the identifier (we already have so many different "keys" ;-) Here, we still stick with identifier: https://github.com/coniks-sys/coniks-go/pull/88/files#diff-7e689735c327b0bd737cd6e956e20b4aR4
Anyways, choose what you prefer. I just think it should be consistent.